### ISSUE TITLE
Table Of Contents Not Sticking

### ISSUE THREAD

**Issue by tommyc38** [angular/material.angular.io#916]
If you open any component that has a TOC and scroll, the TOC won't stick.  

The reason this is happening is because overflow is set to hidden in component-sidenav/component-sidenav.scss:

```
.docs-component-sidenav-inner-content {
  display: flex;
  flex-direction: row;

  // The rule will match the element following the router-outlet which will be the routed component.
  router-outlet + * {
    flex-grow: 1;
    overflow: hidden;
  }
}
``` 

This is also causing issues with the active link status on the links in the TOC (although it doesn't really matter since you can't see it anyway when its scrolled).  

If you remove the overflow, everything works properly.

**Comment by Splaktar** [comment]
This seems related to https://github.com/angular/material.angular.io/issues/285, but that one is only for Edge/IE11. What browser were you using when you saw this behavior?

**Comment by Splaktar** [comment]
Closed #285 as a duplicate of https://github.com/angular/material.angular.io/issues/257. Going to do the same with this one and try to consolidate things there and get this fixed.
